tests/controllers/case-audit-controller.test.js

```diff
@@ -15,57 +14,6 @@ const CaseAuditController = createCaseAuditController(MockCaseAudit);
 
 afterEach(() => jest.clearAllMocks());
 
-test('getContactIdsFromCaseAudits', () => {
```

Comment:
The code this is testing was moved and made private. Normally I would add tests elsewhere to ensure the same level of coverage is maintained, but it's a POC, so meh